test/support/test_repo/migrations/20221102120000_create_deferred_rabbits.exs

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+
+defmodule Carbonite.TestRepo.Migrations.CreateDeferredRabbits do
+  use Ecto.Migration
+
+  def change do
+    create table(:deferred_rabbits) do
+      add(:name, :string)
+      add(:age, :integer)
+    end
+
+    Carbonite.Migrations.create_trigger(:deferred_rabbits, initially: "deferred")
```

Comment:
![](https://media.giphy.com/media/VcDqJibr7niDK/giphy.gif)

Reply:
![](https://media.giphy.com/media/NKeVGRQ8Uj7Da/giphy.gif)

rabbits are cute

btw when you translate "Hase" and "Kaninchen" through google, it both comes out "Rabbit" :shrug:

Reply:
Und Google so:

"kaninchen englisch" -> "rabbits"
"hase" -> "hare"
"Hase" -> "Rabbit"